templates/zerver/help/automatic-theme-detection.md

```diff
@@ -0,0 +1,17 @@
+# Automatic Theme Detection
```

Comment:
"Automatic theme detection" is our style -- we use "Sentence case", not "Title Case" in all Zulip strings.